.github/workflows/release.yml

```diff
@@ -0,0 +1,124 @@
+name: Release Packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: "Type of release"
+        required: true
+        default: "patch"
+        type: choice
+        options:
```

Comment:
need to double check this config here actually - wouldn't this be specified in the changeset file?